### PR TITLE
Bug fix: Enable choosing a Dropdown option with the keyboard

### PR DESCRIPTION
**Pull request by mellogarrett**
## Description

Enable choosing a Dropdown option with the keyboard only

## Type of Change 

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: no-config-components/dropdown-question/index.tsx [hunk 1]
```diff
@@ -109,9 +115,7 @@ export const DropdownQuestionComponent = ({ question, styles, className }: Dropd
                 key={`chip-${choice.value}${choice.key}`}
                 label={choice.display || choice.value}
                 onDelete={() => {
-                  if (choice) {
-                    dispatch(updateMultiOptionAction(choice.value.toString(), questionKey));
-                  }
```
Comment (MarkDKoslow) [inline no-config-components/dropdown-question/index.tsx:114]: Why this change?

Comment (mellogarrett) [reply no-config-components/dropdown-question/index.tsx:114]: Choice is always defined here

Comment (MarkDKoslow) [reply no-config-components/dropdown-question/index.tsx:114]: Oh I see, thought this was deleting from the search text not the chip
